### ISSUE TITLE
`make up-standalone` does not work

### ISSUE THREAD

**Issue by zhenik** [fredrikhgrelland/terraform-nomad-presto#80]
## Current behavior
Fail at resolving variables

```
TASK [Terraform presto standalone] *********************************************
fatal: [default]: FAILED! => {
    "changed": false
}

MSG:

Failed to validate Terraform configuration files:

Error: Unsupported argument

  on main.tf line 37, in module "presto":
  37:   shared_secret_provider = local.presto.shared_secret_provider

An argument named "shared_secret_provider" is not expected here.


Error: Unsupported argument

  on main.tf line 39, in module "presto":
  39:   shared_secret_vault = {

An argument named "shared_secret_vault" is not expected here.



PLAY RECAP *********************************************************************
default                    : ok=9    changed=1    unreachable=0    failed=1    skipped=12   rescued=0    ignored=0   

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
make: *** [up-standalone] Error 1
```

## Expected behaviour
Up and running a standalone example

## How to reproduce?
```
make clean
make up-standalone
```
## Suggestion(s)/solution(s) [Optional]
Setup proper variables

## Checklist (after created issue)
- [X] Added label(s)
- [X] Added to project
- [X] Added to milestone

**Comment by pdmthorsrud** [comment]
Does it fail as soon as the terraform task begins, or does it take a while?

**Comment by zhenik** [comment]
> Does it fail as soon as the terraform task begins, or does it take a while?
When the terraform task begins, due to `main.tf` refer to non-existing variables.

IMHO. I want to mention, that I do not like using `locals` in examples. In my opinion, examples (`main.tf` files) should be straight forward and self-efficient, without defining additional variables in `locals`

**Comment by pdmthorsrud** [comment]
Right. I have no clue how I didn't catch this when making the standalone version earlier. Very weird

**Comment by pdmthorsrud** [comment]
> IMHO. I want to mention, that I do not like using `locals` in examples. In my opinion, examples (`main.tf` files) should be straight forward and self-efficient, without defining additional variables in `locals`

Are you saying you'd like it to be a different `variables.tf` instead? Or that we should use another approach?

**Comment by zhenik** [comment]
Not different variables, just remove this block of code https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/example/presto_standalone/main.tf#L1-L24

**Comment by claesgill** [comment]
I fixed this in #79 . Think we can close this issue!

**Comment by pdmthorsrud** [comment]
> I fixed this in #79 . Think we can close this issue!

Oh, nice! Sorry I'm not sure how that snuck its way into my PR 😅

**Comment by claesgill** [comment]
> > I fixed this in #79 . Think we can close this issue!
> 
> Oh, nice! Sorry I'm not sure how that snuck its way into my PR 😅

Probably because of the tests only tested the cluster example 😅
